app/package.json

```diff
@@ -65,9 +58,7 @@
     },
     "win": {
       "target": [
-        "zip",
-        "msi",
```

Comment:
Another semi related comment, but msi is deprecated from around 2018/2019 I think.